externalAPI/tradeshiftAPI/supplierAPI.go

```diff
@@ -95,27 +94,30 @@ func (t *TradeshiftAPI) SearchOffer(name string) (map[string]interface{}, error)
 	return r, err
 }
 
+func buildAdvancedSearchValue(name string) string {
+	return fmt.Sprintf("{\"name\":\"%v\"}", url.QueryEscape(name))
```

Comment:
QueryEscape method not applicable here. Value should be escaped as JSON.

Reply:
fixed also =) set up values as json